app/src/main/java/com/benoitquenaudon/daggerbundleinjectiondemo/ui/output/OutputPresenter.java

```diff
@@ -3,12 +3,11 @@
 import com.benoitquenaudon.daggerbundleinjectiondemo.injection.SomeInput;
 import javax.inject.Inject;
 
-public class OutputPresenter {
+class OutputPresenter {
   @Inject @SomeInput String inputText;
-  private final OutputActivity activity;
+  @Inject OutputActivity activity;
```

Comment:
I can revert it, and add the string into the constr as well. Do you know of any difference between the two? I only realized you could inject fields into constr injected objects from this code actually. So I'm clueless on it.